### PR TITLE
Add data preprocessing, regularization, learning rate decay, and end conditions

### PR DESCRIPTION
**Pull request by xuefeng-xu**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](https://github.com/primihub/community#development-workflow) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] Please provide a readable PR title.

### Description
Please describe what have you done in this pull request.
- data preprocessing: refer to sklearn MinMaxScaler (default)
- l2 regularization with alpha = 0.0001 by default
- 'optimal' learning rate: refer to sklearn SGDClassifier
- end condition: 'max_iter' = 1000, stop if no changes in accuracy greater than 5 iterations by default

### CODE REVIEW

**Comment by xuefeng-xu** [comment]
Add feature selection, which can be set in `feature_names` in config

File: python/primihub/FL/model/logistic_regression/homo_lr.py [hunk 1]
```diff
@@ -9,15 +9,157 @@
 import pickle
 from primihub.FL.model.logistic_regression.vfl.evaluation_lr import evaluator
 
-from primihub.FL.model.logistic_regression.homo_lr_base import LRModel
```
[the remark this answers is not in the document]
Comment (xuefeng-xu) [reply python/primihub/FL/model/logistic_regression/homo_lr.py:12]: Add feature selection, which can be set in feature_names in config

**Review by xujiangyu — APPROVED** [review]
